CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @cloud-gov/platform-ops @cloud-gov/customer-success-squad
```

Comment:
@cloud-gov/customer-success-squad doesn't have access to this repo so needs to be removed

Reply:
fixed